modules/adding-a-model-server-to-your-data-science-project.adoc

```diff
@@ -21,7 +21,7 @@ ifndef::upstream[]
 endif::[]
 ifdef::upstream[]
 * If you want to use a custom model-serving runtime for your model server, you have added and enabled the runtime. See link:{odhdocshome}/working-on-data-science-projects/#adding-a-custom-model-serving-runtime_nb-server[Adding a custom model-serving runtime].
-* If you want to use graphics processing units (GPUs) with your model server, you have enabled GPU support. This includes installing the Node Feature Discovery and GPU Operators. For more information, see https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html[NVIDIA GPU Operator on Red Hat OpenShift Container Platform] in the NVIDIA documentation.
+* If you want to use graphics processing units (GPUs) with your model server, you have enabled GPU support. This includes installing the Node Feature Discovery and GPU Operators. For more information, see https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html[NVIDIA GPU Operator on {org-name} OpenShift Container Platform] in the NVIDIA documentation.
```

Comment:
For accuracy, it's the NVIDIA GPU Operator only, not the "GPU Operator".

Reply:
I think this should be rolled into a new ticket with your concerns here https://github.com/opendatahub-io/opendatahub-documentation/pull/68#discussion_r1389178875.